### PR TITLE
Clean up styles

### PR DESCRIPTION
**Pull request by devinjameson**
Why: prior to this commit, we had many styles that were either unused or
not implemented properly.

This commit: cleans up our styles.

### CODE REVIEW

File: src/SelfAssessment/AssessmentStart.tsx [hunk 1]
```diff
@@ -50,3 +50,4 @@ const style = StyleSheet.create({
 })
 
 export default AssessmentStart
+
```
Comment (aledustet) [inline src/SelfAssessment/AssessmentStart.tsx:53]: This might be failing the validation

File: src/More/Menu.tsx [hunk 2]
```diff
@@ -166,8 +166,8 @@ const style = StyleSheet.create({
     ...Typography.buttonTextLight,
   },
   icon: {
-    maxWidth: Spacing.icon,
-    maxHeight: Spacing.icon,
+    maxWidth: Iconography.small,
```
Comment (johnschoeman) [inline src/More/Menu.tsx:169]: 💯

File: src/styles/forms.ts [hunk 3]
```diff
@@ -5,28 +5,29 @@ import * as Spacing from "./spacing"
...
+  backgroundColor: Colors.primaryBackground,
   borderRadius: Outlines.baseBorderRadius,
   borderColor: Colors.formInputBorder,
-  borderWidth: 2,
+  borderWidth: Outlines.thin,
   justifyContent: "center",
   fontSize: Typography.medium,
-  // Just using padding breaks vertical padding on multiline text inputs
+
+  // Just using padding below breaks vertical padding on multiline text inputs
```
Comment (johnschoeman) [inline src/styles/forms.ts:18]: thoughts on removing this comment?

File: src/SelfAssessment/AssessmentStart.tsx (hunk 1 above)
Comment (johnschoeman) [reply src/SelfAssessment/AssessmentStart.tsx]: you can run `yarn format` to run prettier for every file.

Comment (devinjameson) [reply src/SelfAssessment/AssessmentStart.tsx]: Yep!

**Comment by devinjameson** [comment]
> I love this pr so much.

<3

File: src/styles/forms.ts (hunk 3 above)
Comment (devinjameson) [reply src/styles/forms.ts:18]: I'm down!

**Pull request merged**